src/lib/api/org/index.ts

```diff
@@ -275,3 +276,81 @@ function convertToSnykProject(projectData: v3ProjectData[]) : SnykProject[] {
   return projects
 }
 
+export async function listTargets(
+  requestManager: requestsManager,
+  orgId: string,
+): Promise<TargetsResponse> {
+  getApiToken();
+  getSnykHost();
+  debug(`Listing all targets for org: ${orgId}`);
+
+  if (!orgId) {
+    throw new Error(
+      `Missing required parameters. Please ensure you have set: orgId, settings.
+        \nFor more information see: https://apidocs.snyk.io/?version=2022-09-15%7Ebeta#get-/orgs/-org_id-/targets`,
+    );
+  }
+  
+  const targets = await listAllSnykTargetREST(requestManager, orgId)
+
+  return {targets};
+}
+
+export async function listAllSnykTargetREST(requestManager: requestsManager,
```

Comment:
missing all query parameters here, we will need to filter by `origin` and probably also by empty. Please implement sending `isEmpty` , `origin` and `limit` for now and type the rest so we can choose to send it later.

<img width="1063" alt="CleanShot 2022-10-10 at 11 34 50@2x" src="https://user-images.githubusercontent.com/2911613/194847432-3456eb5b-3c8c-4303-b4f3-4a7035154af7.png">